oeis.py

```diff
@@ -170,6 +170,23 @@ def A000040(start: int = 0, limit: int = 20) -> Collection[int]:
 
 
 @oeis
+def A023811(start, limit):
+    "Largest metadrome (number with digits in strict ascending order) in base n."
+
+    def largest_metadrome(n):
+        result = 0
+        for i, j in enumerate(range(n - 2, -1, -1), start=1):
+            result += i * n ** j
+        return result
+
+    tab = []
+    for n in range(start, start + limit):
+        tab.append(largest_metadrome(n))
+    return tab
+
+
+@oeis
+
```

Comment:
A blank line is not pretty here, and `black` from `Travis` is asking for you to remove it.